spec/lambda_function_failed_spec.rb

```diff
@@ -47,8 +47,8 @@
     end
 
     describe(:error) do
-      it 'returns a generic Pre flight checks failed error message' do
-        expect(deployment.error).to start_with('Pre flight checks failed:')
+      it 'returns full preflight failure details' do
+        expect(deployment.error == '{"RequiredParameters":"FAILED","CommitCheck":"PASSED","ScheduleCheck":"PASSED","ForwardDeployCheck":"PASSED"}')
```

Comment:
This currently doesn't catch anything if you break the code. It should be

 ```suggestion
        expect(deployment.error).to eq('{"RequiredParameters":"FAILED","CommitCheck":"PASSED","ScheduleCheck":"PASSED","ForwardDeployCheck":"PASSED"}')

```

Reply:
Resolved by merge.